saskatoon/saskatoon/urls.py

```diff
@@ -1,5 +1,5 @@
 from django.contrib import admin
-from django.urls import path, include
+from django.urls import path, include, re_path
```

Comment:
I don't think we should use both `re_path` and `path` if `path` can handled all our cases. I'm not against `re_path`, it's just that if it's not necessary, we should stick with what we already use, in my opinion.

Reply:
done